docs/transactions/iti41.md

```diff
@@ -2,40 +2,170 @@
 
-When publishing a CH-EMED-EPR documents, all references (from the key elements) shall be known to the eMedication service. All non-key elements are ignored by the eMedication service. 
+### Referencing external documents
+When publishing CH-EMED-EPR documents, all references (from the key elements) shall be known to the eMedication service. All non-key elements are ignored by the eMedication service. 
 
 This implies that the metadata cannot contain [Reference ids](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.28). This option is not supported, and submission of a reference to an existing document is forbidden.
 
+### Rules for each type of document
+#### MTP
+* Treatment duration (defined in [dosage](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-dosage.html)) should be within the period of validity of the document (defined by [`DocumentEntry.serviceStatTime`](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.19) and [`DocumentEntry.serviceStopTime`](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.20) metadata).
+#### PRE
+* Effective time of [`documentationOf`](#FIXME) and [`serviceEvent`](#FIXME) has to be aligned with [`DocumentEntry.serviceStatTime`](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.19) and [`DocumentEntry.serviceStopTime`](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.20).
+* Treatment duration (defined in [dosage](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-dosage.html)) should be within the period of validity of the document (defined by [`DocumentEntry.serviceStatTime`](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.19) and [`DocumentEntry.serviceStopTime`](https://profiles.ihe.net/ITI/TF/Volume3/ch-4.2.html#4.2.3.2.20) metadata).
+* Each PRE item should refer to an MTP item that must be a [available](#FIXME) [approved](#FIXME) [valid](#FIXME) [active](#FIXME) and for the same patient
+    * Not blocking issues should not prevent publication but should be notified to service administration to address the resulting ad-hoc issue    
+* [Validity of document](#FIXME) has to be after the [date of the PRE document](#FIXME) and the [start date of duratio](#FIXME)n of treatment of parent MTP 
+* [Code of medication](#FIXME) should match the code of medication of referenced MTP.      
```

Comment:
`CHEMEDEPRMedication.code`
I don't think this is checked at the moment, I should verify this.